app.js

```diff
@@ -0,0 +1,58 @@
+if (localStorage.getItem('Added Books') == null) {
+  localStorage.setItem('Added Books', JSON.stringify([]));
+}
+
+const storeData = JSON.parse(localStorage.getItem('Added Books'));
+
+function updateData() {
+  localStorage.setItem('Added Books', JSON.stringify(storeData));
+}
+
+function createBook(arr) {
+  let books = '';
+  for (let i = 0; i < arr.length; i += 1) {
+    books += `
+      <p>${arr[i].title}</p>
+      <p>${arr[i].author}</p>
+      <button onclick = "removeBook(${i})">Remove</button>
+      <hr/>
+    `;
+  }
+  return books;
+}
+
+function displayBooks() {
+  const listOfBooks = document.querySelector('.container');
+  listOfBooks.innerHTML = `
+    <ul>
+      ${createBook(storeData)}
+    </ul>
+  `;
+}
+
+function addNewData(bookTitle, bookAuthor) {
+  const Book = {
+    title: bookTitle,
+    author: bookAuthor,
+  };
+  storeData.push(Book);
```

Comment:
- [ ] [OPTIONAL] I'd suggest you use the `concat()` method instead of pushing directly to the array as it's considered bad practice and could cause data inconsistency when you directly manipulate the array (although it might seem very mundane here as your project grows this will definitely come in handy)
- You could refactor it to something like this:
 ```
storeData = storeData.concat(Book);
```